gpslogger/src/main/res/xml/pref_general.xml

```diff
@@ -56,15 +54,19 @@
         android:key="debuglogtoemail"
         android:enabled="true"/>
 
-
     <com.afollestad.materialdialogs.prefs.MaterialListPreference
         android:key="changelanguage"
         android:title="@string/change_language_title"
         android:summary="@string/change_language_summary"
         android:icon="@drawable/translate"
         />
 
-
+    <Preference
+        android:title="@string/reset_app_title"
+        android:summary="@string/reset_app_summary"
+        android:icon="@android:drawable/ic_delete"
+        android:key="resetapp"
+        android:enabled="true"/>
 
```

Comment:
New feature - reset the app.  It will delete storage and cache and any local files, and revoke permissions and terminate the app.